assets/travis-worker/travis-worker@.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
Do we still need the previous `worker.service` file then or does this replace it?

Reply:
We don't need the previous worker service file on GCE, but it _is_ in use elsewhere 👍